modules/upgrade/proc-upgrade-rhdh-helm.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-upgrade-rhdh-helm_{context}"]
+
+= Upgrading the {product} Helm Chart
+
+You can upgrade to a later version of {product} in {ocp-short} by using either the web console or the CLI.
```

Comment:
Perhaps more a point of clarification: is the Helm chart the same as the RHDH? So if you update the Helm chart by either the web console or the CLI, you are updating Red Hat Developer Hub, correct? 

The title suggests the user is only upgrading the Helm chart. The first sentence suggests the users is upgrading Red Hat Developer Hub, so wanted to check if there is a difference.

Reply:
I agree this is confusing. When I asked the SMEs this same question (specifically for the Operator), the answer I received was along the lines of _yes, you update the Operator to update RHDH_.